### DISCUSSION TITLE
Add Cash App p2p payment method?

### DISCUSSION THREAD

**Discussion by wildwisher** [bisq-network/bisq#6607] [Ideas]
Hello, Could Cash App be added as a pymtn method for USA?  super easy to use. Simple bank account link, send $$ instantly.
Somewhat like revolut, but dont have to wait like 2 days for funds from bank to Revolut. 
Also revolut sucks, shutdown my account, no reasoning. (not surprised though, woke banksters)
Zelle is not good either since for many, like i, can only do $500 transaction a month.  b.s.

**Comment by pazza83** [comment]
It has been added before but was removed from Bisq due to chargeback issues.
